### PR TITLE
feat: return `toVisit` from `cleanDepositQueue` method

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

File: src/lib/QueueLib.sol [hunk 1]
```diff
@@ -136,7 +136,7 @@ library QueueLib {
         Queue storage self,
         mapping(uint256 => NodeOperator) storage nodeOperators,
         uint256 maxItems
-    ) external returns (uint256 toRemove) {
+    ) external returns (uint256 toRemove, uint256 toVisit) {
```
Comment (dgusakov) [inline src/lib/QueueLib.sol:139]:
```suggestion
    ) external returns (uint256 removed, uint256 lastRemovedBatch) {
```

Comment (madlabman) [reply src/lib/QueueLib.sol:139]: It's not a batch, it's rather the batch's 1-based index.

Comment (madlabman) [reply src/lib/QueueLib.sol:139]: `lastRemovedAtDepth` maybe?

File: src/CSModule.sol [hunk 2]
```diff
@@ -1322,9 +1322,15 @@ contract CSModule is
     /// @notice Clean the deposit queue from batches with no depositable keys
     /// @dev Use **eth_call** to check how many items will be removed
     /// @param maxItems How many queue items to review
-    /// @return Number of the deposit data removed from the queue
-    function cleanDepositQueue(uint256 maxItems) external returns (uint256) {
-        return depositQueue.clean(_nodeOperators, maxItems);
+    /// @return removed Count of batches to be removed by visiting `maxItems` batches
+    /// @return lastRemovedAtDepth The value to use as `maxItems` to remove `removed` batches
```
Comment (dgusakov) [inline src/CSModule.sol:1326]:
```suggestion
    /// @return lastRemovedAtDepth The value to use as `maxItems` to remove `removed` batches if the static call of the method was used
```

**Review by dgusakov — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**